### PR TITLE
Add task to force server to boot from disk using ipmi

### PR DESCRIPTION
**Pull request by nucklehead**

### CODE REVIEW

**Comment by rackhdmirror** [comment]
@RackHD/corecommitters please review.

**Comment by anhou** [comment]
@nucklehead Need some simple unittests here, you could refer to other unittest cases under 'spec' directory. :+1: after with unittests

File: lib/task-data/tasks/set-boot-disk.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Set Node Diskboot',
+    injectableName: 'Task.Obm.Node.DiskBoot',
+    implementsTask: 'Task.Base.Obm.Node',
```
Comment (yyscamper) [inline lib/task-data/tasks/set-boot-disk.js]: Refer to `setPxeBoot` (https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/set-boot-pxe.js#L9) , please specify the `schemaRef`

**Comment by yyscamper** [comment]
jenkins: ok to test

**Comment by rackhdmirror** [comment]
Build finished.

File: lib/task-data/tasks/set-boot-disk.js (hunk 1 above)
Comment (nucklehead) [inline lib/task-data/tasks/set-boot-disk.js]: Thanks. I will add the schemaRef

File: lib/task-data/tasks/set-boot-disk.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2015, EMC, Inc.
```
Comment (nucklehead) [inline lib/task-data/tasks/set-boot-disk.js]: Thanks. This was created sometime ago and I am doing the PR now. I will update it

File: lib/task-data/tasks/set-boot-disk.js (hunk 1 above)
Comment (nucklehead) [inline lib/task-data/tasks/set-boot-disk.js]: This is added

File: lib/task-data/schemas/obm-control.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
                 "powerStatus",
                 "reboot",
                 "reset",
+                "setBootDisk",
```
Comment (nucklehead) [inline lib/task-data/schemas/obm-control.json]: @yyscamper

**Comment by anhou** [comment]
@nucklehead  codes have conflicts now, please try to rebase them.

**Comment by yyscamper** [comment]
@nucklehead : pleas resolve the conflict, so that we can merge it.

**Comment by nucklehead** [comment]
@yyscamper all conflicts are resolved

**Comment by yyscamper** [comment]
jenkins: test this please

**Pull request merged**